src/main/java/uk/gov/gchq/palisade/resource/request/AddResourceRequest.java

```diff
@@ -17,6 +17,7 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 
+import uk.gov.gchq.palisade.Generated;
 import uk.gov.gchq.palisade.RequestId;
 import uk.gov.gchq.palisade.ToStringBuilder;
 import uk.gov.gchq.palisade.exception.ForbiddenException;
```

Comment:
use setters for the methods below

Reply:
The AddResourceRequest methods (lines 48-58) need to use setters and not check nulls